src/MangaStream.ts

```diff
@@ -345,7 +350,7 @@ export abstract class MangaStream implements ChapterProviding, HomePageSectionsP
 
     async getSearchTags(): Promise<TagSection[]> {
         const request = App.createRequest({
-            url: `${this.baseUrl}/${this.directoryPath}/`,
+            url: `${this.baseUrl}/${this.filterPath}/`,
```

Comment:
What's the purpose for this and why should it be added? Has not been needed before?

Reply:
> I'm trying to seperate values to make it future-proof. In hindsight, I don't think it'll be useful anymore and will remove it in a subsequent patch commit.